contributing_to_docs/doc_guidelines.adoc

```diff
@@ -1318,12 +1315,20 @@ a|
 `<project>`
 
 `<deployment>`
+
+`<install_mode_value>`
 ....
 
 a|
 > Use the following command to roll back a Deployment, specifying the Deployment name:
 >
 > `oc rollback <deployment>`
+>
+> &nbsp;
+>
+> Apply the new configuration file:
+>
+> `oc apply -f <path_to_configuration_file>/<filename>.yaml`
```

Comment:
Nitpicky, but shouldn't it be <file_name> two words?

Reply:
I thought so too, but the existing example had it as one so I went with that. I can certainly change both to `file_name` though if that's inline with the ISG :smiley:

Reply:
Mentioning ISG guidance here because it's surprising:

**file name** 
Write as shown (two words) unless you are using it as a variable.

**filename**
Use only as a variable. Otherwise, use "file name".

Reply:
Thanks! I wasn't aware of that exception